ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -141,31 +137,33 @@ export default function EditGasDisplay({
             </Typography>
           </div>
         )}
-        {!requireDappAcknowledgement && (
-          <RadioGroup
-            name="gas-recommendation"
-            options={[
-              {
-                value: GAS_RECOMMENDATIONS.LOW,
-                label: t('editGasLow'),
-                recommended: defaultEstimateToUse === GAS_RECOMMENDATIONS.LOW,
-              },
-              {
-                value: GAS_RECOMMENDATIONS.MEDIUM,
-                label: t('editGasMedium'),
-                recommended:
-                  defaultEstimateToUse === GAS_RECOMMENDATIONS.MEDIUM,
-              },
-              {
-                value: GAS_RECOMMENDATIONS.HIGH,
-                label: t('editGasHigh'),
-                recommended: defaultEstimateToUse === GAS_RECOMMENDATIONS.HIGH,
-              },
-            ]}
-            selectedValue={estimateToUse}
-            onChange={setEstimateToUse}
-          />
-        )}
+        {!requireDappAcknowledgement &&
+          ![EDIT_GAS_MODES.SPEED_UP, EDIT_GAS_MODES.CANCEL].includes(mode) && (
```

Comment:
No recommendations for speed up or cancellation?

Reply:
yeah i believe we should be showing recommendations for these. Previously we would just default the fields to 110% of the previous value but there was no warning for going under that. Now we're going to use present estimations but warn if the value is less than 110% of previous value. That work doesn't have to be done here but I'm also curious about this logic change.

Reply:
I can restore it, but they aren't present in the Figma file:  https://www.figma.com/file/NXvUKcZ0Ocu1GPcT7VxRMx/1559---Gas-Controls?node-id=643%3A5

Reply:
talked on the sync up this morning and they do not get recommendations right now we may restore at a later time.